src/main/java/org/folio/consortia/service/impl/PublicationServiceImpl.java

```diff
@@ -293,4 +293,18 @@ public PublicationResultCollection getPublicationResults(UUID consortiumId, UUID
       .totalRecords(ptrEntities.getSize());
   }
 
+  @Override
```

Comment:
requires `@Transactional` annotation in order to delete records from two tables transactionally

Reply:
Thank you, added.